### ISSUE TITLE
question about dataset

### ISSUE THREAD

**Issue by changykang** [yaoyao-liu/meta-transfer-learning#53]
hi,i download the dataset from the link, but they donot include lable files, can you help me solve this problem. thank you.

**Comment by yaoyao-liu** [comment]
Thanks for your interest in our work.

You don't need the label files. The images for one class are saved in a corresponding folder. 
You may use [the provided dataloader](https://github.com/yaoyao-liu/meta-transfer-learning/blob/main/pytorch/dataloader/dataset_loader.py) to generate episodes for few-shot learning.

**Comment by changykang** [comment]
can i directly run the tensorflow file?

**Comment by changykang** [comment]
thank you solve my problem,and can i directly run the run_experiment.py in tensorflow?

**Comment by yaoyao-liu** [comment]
Yes. After you install all the requirements and put the data in the proper directory, you may run the command line following the README.md file.

**Comment by changykang** [comment]
ok, thank you reply, when i install tensorflow-gpu=1.3.0 under the python=2.7, it shown packagenotfounderror, can you help me.

**Comment by yaoyao-liu** [comment]
If the packages can not be directly downloaded, you may build TensorFlow 1.3.0 according to [this link](https://www.tensorflow.org/install/source#tensorflow_1x).

**Comment by changykang** [comment]
Thank you for solving my problem patiently

**Comment by changykang** [comment]
i have another question about the dataset, what is the dataset in the meta-train and meta-test, and how did you split the support set and query set in the miniImageNet dataset? thank you

**Comment by yaoyao-liu** [comment]
The splits of miniImageNet are available [here](https://github.com/yaoyao-liu/mini-imagenet-tools/tree/main/mini_imagenet_split).
There are two different splits for miniImageNet. Most papers follow the split provided by [Vinyals et al.](https://proceedings.neurips.cc/paper/2016/file/90e1357833654983612fb05e3ec9148c-Paper.pdf)

During the meta-training phase, the support and query sets of each episode/task are randomly sampled from the training set of miniImageNet. During the meta-test phase, the support and query sets of each episode/task are randomly sampled from the test set of miniImageNet. You may easily find the generation protocol in any few-shot learning paper, e.g., [MAML](https://arxiv.org/pdf/1703.03400.pdf).

**Comment by yaoyao-liu** [comment]
During the pre-train phase, we train the encoder on a 64-way classification task using the meta-train set.

During the meta-train phase, we train the meta model on many 5-way small classification tasks generated from the meta-train set.

During the meta-test phase, we evaluate the model on many 5-way small classification tasks generated from the meta-test set.
